banksystem/accounts/views.py

```diff
@@ -1,5 +1,9 @@
-from django.shortcuts import render, HttpResponse
+from django.shortcuts import render
 
+from .models import Account
 
-def index(request):
-    return HttpResponse('Welcome to the Account')
+
+def accounts(request):
+    data = Account.objects.all()
```

Comment:
study django ORM and `filter` method of queryset

Reply:
we need to return only the accounts of user who has logged in. not all accounts!